lib/aur-vercmp

```diff
@@ -68,7 +68,7 @@ trap_exit() {
 }
 
 usage() {
-    plain "usage: $argv0 [-p path] [-acq]" >&2
+    plain 'usage: %s [-p path] [-acq]' "$argv0" >&2
```

Comment:
I guess if we're aiming for consistency and add `%s` everywhere (less shellcheck complaints?), the same should be done for `error`/`warning`

Reply:
> I guess if we're aiming for consistency and add %s everywhere (less shellcheck complaints?)

I don't think shellcheck complains about having the format:
```diff
-    plain "usage: $argv0 [-p path] [-acq]" >&2
```

I just picked one and went with it. If you prefer the this, I can easily change all to this format. I only care about being consistent across the board.

> the same should be done for error/warning
Agreed, The original intent for this PR was to make `usage()` consistent. So I restricted to changes to usage.

We can rename this PR something else and add commits fixing there rest.

Reply:
Works for me as is. We can make the changes to plain et al in a second PR.